Makefile

```diff
@@ -485,3 +490,15 @@ release-bundle: cmd/konvoy-image-wrapper/image/konvoy-image-builder.tar.gz
 	$(MAKE) GOOS=linux release-bundle-GOOS
 	$(MAKE) GOOS=windows release-bundle-GOOS
 	$(MAKE) GOOS=darwin release-bundle-GOOS
+
+.PHONY: create-image-list
+create-image-list:
+	@rm -f images.out
+	@ansible-playbook ./ansible/list-images.yaml -e="@./overrides/image-list.yaml"
```

Comment:
I don't love this. It's assuming that some version of ansible-playbook in my path is the same as your path. 

We have a docker container for developing, do you think we can fix this in a future version? 

Non blocking comment though.

Reply:
I guess we can do devkit.run WHAT=create-image-list